parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ColumnChunkPageWriteStore.java

```diff
@@ -215,15 +226,18 @@ public void writeDictionaryPage(DictionaryPage dictionaryPage) throws IOExceptio
     public String memUsageString(String prefix) {
       return buf.memUsageString(prefix + " ColumnChunkPageWriter");
     }
+
   }
 
   private final Map<ColumnDescriptor, ColumnChunkPageWriter> writers = new HashMap<ColumnDescriptor, ColumnChunkPageWriter>();
   private final MessageType schema;
 
-  public ColumnChunkPageWriteStore(BytesCompressor compressor, MessageType schema, int pageSize) {
+  // TODO - look back at this, an allocator was being passed here in the ByteBuffer changes
+  // See the comment at the ColumnChunkPageWriter constructor above
```

Comment:
For the reasons given above, it doesn't look like there is an available interface for refactoring this anyway. I'll remove the TODO.